content/tutorials/webrtc/basics/en/index.html

```diff
@@ -165,6 +169,8 @@ <h2 id="toc-mediastream">MediaStream (aka getUserMedia)</h2>
 
 <p>Likewise on pages using HTTPS: permission only has to be granted once for for <code>getUserMedia()</code> (in Chrome at least). First time around, an Always Allow button is displayed in the browser's <a href="http://dev.chromium.org/user-experience/infobars" title="Chromium information about the browser infobar">infobar</a>.</p>
 
+<p>Note that HTTP access for getUserMedia() will be deprecated soon in Chrome due to it being classified as a <a href="https://sites.google.com/a/chromium.org/dev/Home/chromium-security/deprecating-powerful-features-on-insecure-origins" title="Chrome powerful features">Powerful feature</a>. You can already see a warning when invoked on a HTTP page on Chrome M44.</p>
```

Comment:
I do not know which versions it actually will be deprecated, currently the powerful feature article states 2015.  Will add end of 2015 instead for now.